### PR TITLE
Enable cross compilation

### PR DESCRIPTION
**Pull request by ThomasMarwitzQC**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ThomasMarwitzQC** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
Based on this: https://github.com/nanomsg/nng/blob/master/docs/BUILD_CROSSCOMPILE.adoc I think tool build (nngcat) is disabled when cross compiling, so you should either not test it, or enable it manually in the build flags.

File: recipe/build.sh [hunk 1]
```diff
@@ -11,7 +11,8 @@ if [[ "$target_platform" == osx-arm64 ]]; then
 		-DCMAKE_INSTALL_LIBDIR=lib \
 		-DCMAKE_BUILD_TYPE=Release \
 		-DCMAKE_VERBOSE_MAKEFILE=ON \
-		-DENABLE_TESTING=Off
+		-DENABLE_TESTING=OFF \
+  		-DNNG_TOOLS=ON
```
Comment (pavelzw) [inline recipe/build.sh]:
```suggestion
		-DNNG_TOOLS=ON
```

this is currently mix between spaces and tabs
image

File: recipe/build.sh [hunk 2]
```diff
@@ -22,7 +23,8 @@ else
 		-DCMAKE_INSTALL_PREFIX=$PREFIX \
 		-DCMAKE_INSTALL_LIBDIR=lib \
 		-DCMAKE_BUILD_TYPE=Release \
-		-DCMAKE_VERBOSE_MAKEFILE=ON
+		-DCMAKE_VERBOSE_MAKEFILE=ON \
+  		-DNNG_TOOLS=ON
```
Comment (pavelzw) [inline recipe/build.sh:27]:
```suggestion
		-DNNG_TOOLS=ON
```

**Review by carlodri — APPROVED** [review]

**Pull request merged**